db/schema.rb

```diff
@@ -245,7 +284,7 @@
     t.string "uid"
     t.string "profile_picture_file_name"
     t.string "profile_picture_content_type"
-    t.bigint "profile_picture_file_size"
+    t.integer "profile_picture_file_size"
```

Comment:
related ?

Reply:
I don't think so. I guess when original PR was created, it was an integer. That's why this change. I'll update it right away. @tachyons